applications/services/gui/modules/submenu.c

```diff
@@ -63,7 +64,12 @@ static void submenu_view_draw_callback(Canvas* canvas, void* _model) {
     SubmenuModel* model = _model;
 
     const uint8_t item_height = 16;
-    const uint8_t item_width = 123;
+    uint8_t item_width = 123;
```

Comment:
I think canvas_width will be better option here. Also it doesn't require private methods from canvas_i.h